Package.swift

```diff
@@ -1,6 +1,9 @@
 // swift-tools-version:5.5
 import PackageDescription
 
+// Enable to use bundled Circular Pro fonts (for development only)
```

Comment:
"for development only" can be removed, as we should allow easy bundling for anybody who has a licence to use those fonts (= not only for development), which is also a case of our app.

Maybe it could be "(for licensed usage)" or similar instead?

Similar applies to the Usage section in readme file.

Reply:
Updated, though not sure what you want changed in the readme... the phrasing there is taken directly from the Orbit website (see the link).